locale/hi.yml

```diff
@@ -1273,7 +1273,7 @@ import: "आयात" # ✅
 in: "में" # ✅
 is: "है" # ✅
 lambda: "लैम्ब्डा" # 🤔 "लैम्ब्डा" is just a phonetic translation
-nonlocal: "नॉनलोकल" # ❓ My suggestion : "गैर स्थानीय". "नॉनलोकल" is just a phonetic translation of "nonlocal"
+nonlocal: "नॉनलोकल, गैर स्थानीय" # ❓ My suggestion : "गैर स्थानीय". "नॉनलोकल" is just a phonetic translation of "nonlocal"
```

Comment:
Understanding more of your comment in the previous PR. The initial translation will stay in place and additional reviews will be added behind the comment `#`. Denote your specific translation by separating it by a comma and your name after the last user's review. So for example:

``` yml
nonlocal: "नॉनलोकल # ❓ My suggestion : "गैर स्थानीय". "नॉनलोकल" is just a phonetic translation of "nonlocal", @sshiv5768: गैर स्थानीय
```